build-ioc.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -ex
+
+cp /opt/epics/RELEASE configure/
+cat << EOF > configure/CONFIG_SITE.local
+STATIC_BUILD=YES
+FULL_STATIC_BUILD=YES
+USR_LDFLAGS = -static-pie
+EOF
+
+git config --global --add safe.directory $PWD
+
+make distclean
+make
+make clean
+make -C iocBoot
+
+version=$(git describe)
```

Comment:
I'd also add `--tags` option here to include lightweight tags. By default, it includes only annotated ones.

Reply:
I'd like to standardize on annotated tags, since they can be signed.

But supporting lightweight ones makes sense, specially if we use the script for other projects which we might not have control over.